.github/prlint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "title": [
+        {
+            "pattern": "^(\\[\\d+\\.\\d+\\]\\s{1})?(feat|fix|chore|docs|refactor|style|test)(\\([\\-.@:`a-zA-Z0-9]+\\))?!?:\\s{1}\\S.+\\S$",
+            "message": "PR title does not meet requirement, use Conventional Commits format using one of these categories: feat, fix, chore, docs, refactor, style, test"
```

Comment:
I agree with standardizing PR titles as it makes CHANGELOG.md easier to read.

This message seems difficult to understand what to do for beginners.
The meaning of each category and examples should be shown.

And please add `perf` for performance tuning.